docs/DeveloperDoc.md

```diff
@@ -2,13 +2,15 @@
 
 ## How to render a palette
 
-`Palette.ts` accepts a JSON in a structure demostrated in
-[`src/keyboards/bmw_palette.json`](../src/keyboards/bmw_palette.json). In the
-`cells` object, each cell has a `type` key and a `options` key. The `type`
-value indicates which component should be used to render this cell. The
-`options` contains information to be passed to the component.
+`Palette.ts` constructs a palette based on a JSON file that contains a list of
+of the cells in the palette. An example is found in
```

Comment:
Two "of"s here.  One is at the end of the previous line just before the carriage return, and the other is just after at the beginning of the next line.

Reply:
Good catch. Fixed now. Thanks.